musicrs/model/base.py

```diff
@@ -0,0 +1,33 @@
+""" Base models of the database tables """
+
+from sqlalchemy import Column, Integer, String, TIMESTAMP, func
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
+
+
+class UserProfile(Base):
+    __tablename__ = "user_profile"
+
+    id = Column(Integer, primary_key=True)
+    user_id = Column(String, nullable=False)
+    user_name = Column(String(100), nullable=False)
+    youtube_url = Column(String, nullable=True)
+    audio_encoding = Column(String, nullable=True)
```

Comment:
isn't this supposed to be Array

Reply:
Since you are researching on how to store the encoding, I assigned string for now. I will make change after your confirmation.

Reply:
lets make it varchar for now.

Reply:
Its same thing in sqlalchemy